frontend/src/components/router-head/theme-script.tsx

```diff
@@ -8,6 +8,10 @@ export const ThemeScript = () => {
             .setAttribute('data-theme',
                 localStorage.getItem('${themeStorageKey}') ??
                 (window.matchMedia('(prefers-color-scheme: dark)').matches ? '${DARK_THEME}' : '${LIGHT_THEME}')
-            )`;
+            );
+        if (document.firstElementChild
+            .getAttribute('data-theme') === '${DARK_THEME}')
+                document.firstElementChild.classList.add('dark');
+        `;
```

Comment:
You don't havve to check if the theme is dark twice.
Maybe we should store the theme in const

Reply:
This property allows tailwind to load classes using the format dark:classname, and data-scheme is used to overwrite css variable values, that's why I had to add it

Reply:
Maybe we can optimize this in the future, but I think it's not the big difference right now